client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -162,15 +161,24 @@ button {
  	 * Step Container
  	 */
 	.step-container {
-		--color-accent: #117ac9;
-		--color-accent-60: #0e64a5;
 
 		.form-fieldset {
 			label {
 				text-transform: none;
 			}
 		}
 	}
+
+	// While we don't standardize all Calypso interfaces, we need to override this for onboarding flows #79851
+	.components-button.is-primary {
+		&[disabled],
+		&:disabled,
+		&.disabled {
+			color: #fff;
+			background-color: var(--studio-blue-20);
+			border-color: var(--studio-blue-20);
```

Comment:
Are these meant to be gray?

Fixing this is out of scope for this PR but: this button UX confusing, it looks disabled (sort of) but if you click on it it focuses and submits the form (which fails validation), updating the form changes the color to the normal solid blue.

![Screenshot 2023-07-26 at 12-18-52 Link in Bio — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/109ad29a-0018-45d0-aa31-6210bbfffc4c)

Reply:
https://github.com/Automattic/wp-calypso/pull/79851#issuecomment-1651136523